npm/mount-utils/src/index.ts

```diff
@@ -1,4 +1,66 @@
-import { StyleOptions } from './mount'
+/**
+ * Additional styles to inject into the document.
+ * A component might need 3rd party libraries from CDN,
+ * local CSS files and custom styles.
+ */
+export interface StyleOptions {
+  /**
+   * Creates <link href="..." /> element for each stylesheet
+   * @alias stylesheet
+   */
+  stylesheets: string | string[]
+  /**
+   * Creates <link href="..." /> element for each stylesheet
+   * @alias stylesheets
+   */
+  stylesheet: string | string[]
```

Comment:
Why do all of these have aliases? What is the point?

Reply:
Legacy API design. The styles api needs to be deduped and refined.

Reply:
Can't decide on an API? Why not have them all!